src/frontc/cabs2cil.ml

```diff
@@ -4847,6 +4917,31 @@ and doExp (asconst: bool)   (* This expression is used as a constant *)
 
     | A.EXPR_PATTERN _ -> E.s (E.bug "EXPR_PATTERN in cabs2cil input")
 
+    | A.GENERIC (expr, lst) -> 
+        let get_exp (_, e, _) = e in
+        let exp = get_exp (doExp false (stripParenLocal expr) (AExp None)) in
+        let cil_list = 
+          let rec make_cil_list cabs_l cil_l = match cabs_l with 
+          | [] -> cil_l
+          | (s, e) :: rest -> make_cil_list rest (((doOnlyType s JUSTBASE),get_exp (doExp false (stripParenLocal e) (AExp None))) :: cil_l) 
+          in
+          make_cil_list lst [] 
+        in
+        let exp_typ = typeOf exp in
+        let default_typ = ref voidType in
```

Comment:
This should probably be `ref None` instead, unless somewhere the standard says `default` should be equivalent to `void`.